tools/brownie-example/LICENSE

```diff
@@ -1,6 +1,6 @@
 Hedera Brownie Example
 
-Copyright (C) 2024 Hedera Hashgraph, LLC
+Copyright (C) 2022-2024 Hedera Hashgraph, LLC
```

Comment:
Also please revert these license header changes. See Nana's comment for reference https://github.com/hashgraph/hedera-json-rpc-relay/pull/3350#discussion_r1894545838.